src/main/resources/templates/pps/onlinePaymentUnavailable.html

```diff
@@ -16,72 +16,63 @@
                 </h1>
-                <p class="govuk-body-l">
-                    You can still pay your penalty in other ways.
-                </p>
-                <h2 class="govuk-heading-m">Pay by Bacs</h2>
-                <p class="govuk-body">
-                    You can pay using Bacs payment.
-                </p>
-                <p class="govuk-body">
-                    It usually takes up to 3 working days for your payment to reach Companies House.
-                    Check your bank's transaction limits and processing times before making a payment.
-                </p>
-                <p class="govuk-body">
-                    You'll need to use the Companies House sort code, account number and account name.
-                </p>
-                <div class="govuk-details__text" aria-hidden="true" id="banking-details">
-                    <p th:text="'Sort code: ' + ${@environment.getProperty('chBankSortCode')}"></p>
-                    <p th:text="'Account number: ' + ${@environment.getProperty('chBankAccNumber')}"></p>
-                    <p th:text="'Account name: ' + ${@environment.getProperty('chBankAccName')}"></p>
-                </div>
-                <p class="govuk-body"></p>
-                <h2 class="govuk-heading-m">Paying from an overseas account</h2>
                 <p class="govuk-body">
-                    You can pay through a bank transfer from an overseas account.
+                    You cannot use this online service to pay this penalty.
                 </p>
                 <p class="govuk-body">
-                    You'll need to use Companies House swift code, account number and account name.
+                    Pay by bank transfer instead, using these account details
                 </p>
-                <div class="govuk-details__text" aria-hidden="true" id="overseas-banking-details">
-                    <p class="govuk-body">Swift Code: NW BK GB 2L</p>
-                    <p class="govuk-body">Account Number (IBAN): GB 79 NWBK 607080 10031154 </p>
-                    <p class="govuk-body">Account name: Late Filing Penalties Receipts</p>
+                <div class="govuk-inset-text">Check the details carefully, as they may be different from other
+                    transactions you've made with us.
                 </div>
-                <p class="govuk-body"></p>
-                <p class="govuk-body">Email the LFP finance team at Companies House to tell them you've paid.
-                    Include in your email the:</p>
-                <ul class="govuk-list govuk-list--bullet" id="list-requirements-overseas-payments">
-                    <li>company number</li>
-                    <li>company name</li>
-                    <li>reference number on the penalty notice</li>
-                    <li>amount paid</li>
-                    <li>date paid</li>
+
+                <h2 class="govuk-heading-s">Bacs payment</h2>
```

Comment:
Should this heading be `Bacs payments`?
Also should this heading be `h3` to match prototype WIP - https://sanctions-payment-prototype-0d6d68d6c82d.herokuapp.com/v9/stop-you-need-to-pay-another-way